lib/main.dart

```diff
@@ -68,15 +127,147 @@ class _MyHomePageState extends State<MyHomePage> {
     });
   }
 
+  ///////////////////////////////////////////////////
+  // Функция для вставки видео в плейер
+  @override
+  void initState() {
+    super.initState();
+    // Play a [Media] or [Playlist].
+    player.open(Media('https://user-images.githubusercontent.com/28951144/229373695-22f88f13-d18f-4288-9bf1-c3e078d83722.mp4'));
+  }
+
+  @override
+  void dispose() {
+    player.dispose();
+    super.dispose();
+  }
+
+  void _tellTime() {
+    ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+       content: Text("Video is on ${player.state.position}"),
+     ));
+    //print(player.state.position);
+  }
+
   @override
   Widget build(BuildContext context) {
-    // This method is rerun every time setState is called, for instance as done
-    // by the _incrementCounter method above.
-    //
-    // The Flutter framework has been optimized to make rerunning build methods
-    // fast, so that you can just rebuild anything that needs updating rather
-    // than having to individually change instances of widgets.
+
     return Scaffold(
+      appBar: AppBar( //Верхняя часть с именем
+        // TRY THIS: Try changing the color here to a specific color (to
+        // Colors.amber, perhaps?) and trigger a hot reload to see the AppBar
+        // change color while the other colors stay the same.
+        backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+        // Here we take the value from the MyHomePage object that was created by
+        // the App.build method, and use it to set our appbar title.
+        title: Text(widget.title),
+      ),
+      body: 
+      Row( //Тело, разделённое по колонкам
+      children: [
+        
+        Column(children: [
+          
+          SizedBox( // Коробка под видео
+            width: MediaQuery.sizeOf(context).width * 0.7,
+            //width: MediaQuery.of(context).size.width,
+            height: MediaQuery.of(context).size.width * 0.7 * 9.0 / 16.0,
```

Comment:
Вполне возможно, можно выделить в программе константы типа `RATIO` вместо магических чисел, которые с первого чтения непонятно какую именно функцию выполняют.
```suggestion
            height: MediaQuery.of(context).size.width * 0.7 * RATIO,
```

И потом где-нибудь:
```dart
const RATIO = 9.0 / 16.0;
```